docs/source/algorithms/CrossCorrelate-v1.rst

```diff
@@ -9,20 +9,37 @@
 Description
 -----------
 
-Compute the cross correlation function for a range of spectra with
-respect to a reference spectrum.
+Compute the cross correlation function for a range of spectra with respect to a reference spectrum.
 
-.. math:: c_{fg}(r) = \frac{1}{\sigma_f \sigma_g} \int (f(x) - \mu_f) (g(x+r)-\mu_g) dx
+This is use in powder diffraction experiments when trying to estimate the offset (in number of bin indices) of one spectra with respect to another one.
+The input spectra should be in d-spacing and then interpolate on the X-axis of the reference before calculating the cross correlation in bin index.
```

Comment:
In principle, this should work with momentum transfer too, but I can't think of a practical situation where someone would do this.  Hence the suggested change.

Reply:
In the algorithm, there is a check for the units on the input workspace to be d-spacing. I agree that it should work on Q as well, but haven't ever edited the code so I could try.